### DISCUSSION TITLE
Performance Monitoring

### DISCUSSION THREAD

**Discussion by Belnando** [amidaware/tacticalrmm#707] [Q&A] [answered]
Hi
I have just discovered this RMM and am wondering if it can also do performance monitoring on servers and workstations. Let's imagine that I wanted to know how memory or CPUs are performing.

Thanks for your response. 
Belnando

**Comment by dinger1986** [comment, marked as answer]
You can monitor cpu, ram, disk space usage. Plenty else can be monitored by powershell by scripting that up

**Comment by Belnando** [comment]
Fantastic.... is there documentation on how this can be done?  I am
interested in using these features in this product.

Thanks
Belnando

**Reply by dinger1986** [reply]
Look at automation policy's etc

https://wh1te909.github.io/tacticalrmm/functions/alerting/

**Comment by Belnando** [comment]
Thanks...

Highly appreciated

Belnando

**Comment by bbrendon** [comment]
Just a heads up, Tactical isn't designed to be a performance monitoring tool. It can do it, but if you're really digging into performance, it's not the tool.

**Comment by Belnando** [comment]
Ok, thanks for the clarification. I am not looking for a whole lot of
detail or metrics

Thanks
Belnando
